test/resources/sampleContainer/example1.ttl

```diff
@@ -2,6 +2,8 @@
 @prefix dc: <http://purl.org/dc/elements/1.1/> .
 @prefix ex: <http://example.org/stuff/1.0/> .
 
+</sampleContainer/example*> a :glob.
```

Comment:
@deiu would something like `</sampleContainer/example*> :test "testing serialization base" .` make more sense, instead? 

I don't have a stake in this argument; was just adding the statement at @timbl's suggestion.